.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,32 +1,45 @@
 ---
-name: Bug report
+name: Bug Report
 about: Create a report to help us improve
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
-# Please follow the general troubleshooting steps first:
+Note: If the bug affects multiple games, please open an issue for each game with the game in the title.
 
-- [ ] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and completed successfully the setup wizard.
+# Bug Reports
+
+## Description
+Please replace this line with a brief summary of your issue
+
+## General Troubleshooting
+- [ ] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
 - [ ] I read the ALVR Wikis [here](https://github.com/polygraphene/ALVR/wiki) and [here](https://github.com/alvr-org/ALVR/wiki)
 
-If your issue is about games not working, please open an issue for every game and list the game in the title.
+## Environement
 
-### Bug reports:
+### Hardware
+**Note**: for Linux, an upload to the [`hw-probe`](https://linux-hardware.org/) database is preferred: `hw-probe -all -upload`
 
-Please replace this line with a brief summary of your issue
+**CPU**:
+
+**GPU**:
 
-#### Environement
+**Audio**:
 
-What hardware do you have: CPU, GPU, Audio
-What version of ALVR and SteamVR have you tried. 
-Do you have the latest Windows updates? Run "winver" in Windows 10 to get the version number
+### Installation
+**ALVR Version**:
 
+**SteamVR Version**:
 
-### Features:
+**Install Type**:
+- [ ] Packaged (`exe`, `deb`, `rpm`, etc)
+- [ ] Portable (`zip`)
+- [ ] Source
 
-**Please note by far the quickest way to get a new feature is to file a Pull Request.**
+**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions):
 
-We will consider your request but it may be closed if it's something we're not actively planning to work on.
+# Feature Requests
+The quickest way to get a new feature is to file a pull request; these will be considered, but may be closed if they're something we're not actively planning to work on.
```

Comment:
This also probably doesn't make sense here. This is a form users fill out: have a look at this [example](https://github.com/NixOS/nixpkgs/blob/master/.github/PULL_REQUEST_TEMPLATE.md) ([plaintext](https://raw.githubusercontent.com/NixOS/nixpkgs/master/.github/PULL_REQUEST_TEMPLATE.md) too)

Reply:
```suggestion
<!-- Feature Requests
The quickest way to get a new feature is to file a pull request; these will be considered, but may be closed if they're something we're not actively planning to work on. -->
```